system/HTTP/CURLRequest.php

```diff
@@ -812,7 +821,19 @@ protected function sendRequest(array $curl_options = []): string
 
 		if ($output === false)
 		{
-			throw HTTPException::forCurlError(curl_errno($ch), curl_error($ch));
+			// Check if debug is set to 'true'. If yes send verbose to output. If not throw exceptions.
+			if ($this->config['debug'] === true)
```

Comment:
```suggestion
			if ($this->config['debug'] === true && $this->config['debug'] !== true)
```
I would change it to this condition and only then would throw an exception. Only when `debug` is turned on we want to see the "pure" error from curl.

Reply:
I think there's an extra condition in here, or may an extra `=`? IMO it is fine to specify in the docs that `$config['debug']` can be a string file path OR boolean `true` and then let this throw otherwise.

Reply:
@michalsn yeah you are right. ;)

Reply:
@michalsn Nope, it works fine here.  Print to the screen is triggered only when is set to 'true'. condition in this form `if($this->config['debug])` would be triggered even when there would be string. Now it is triggered only and only with 'true'.